addon-test-support/@ember/test-helpers/dom/-target.ts

```diff
@@ -1,4 +1,4 @@
-type Target = string | Element | Document | Window;
```

Comment:
this is used as type for the first argument in most of the interaction helpers.
  
So, if we were emitting public types, this would be a breaking change. However, under the hood we often just ignore `Document`, and try to do nothing or fail.